.eslintrc.json

```diff
@@ -0,0 +1,54 @@
+{
+    "env": {
+        "es2020": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 2020,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "indent": [
+            "error",
+            2
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "quotes": [
+            "error",
+            "single"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ],
+        "no-console": [
+            "error",
+            {
+                "allow": [
+                    "warn",
+                    "info"
+                ]
+            }
+        ],
+        "curly": [
+            "error",
+            "all"
+        ],
+        "lines-between-class-members": [
```

Comment:
Is there any way to configure this to 3 lines for each class members?

Reply:
According to [this](https://eslint.org/docs/rules/lines-between-class-members), no.

Reply:
x.x aww... that's so unfortunate.